src/components/Controller.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import SearchingPage from "../redux/containers/SearchingPage";
+import Favorites from "../redux/containers/Favorites";
+import Player from "./Player";
+import Menu from "../redux/containers/Menu";
+
+class Controller extends Component {
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
1. Зачем оборачивать Router в Fragment ? 
2. Запись можно сократить, импортируя Fragment
```suggestion
      <Fragment>
```